src/jit/gentree.h

```diff
@@ -3803,16 +3806,17 @@ struct GenTreeCmpXchg : public GenTree
 };
 
 #if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
-struct GenTreeMulLong : public GenTreeOp
+struct GenTreeMultiRegOp : public GenTreeOp
 {
     regNumber gtOtherReg;
 
-    GenTreeMulLong(var_types type, GenTreePtr op1, GenTreePtr op2) : GenTreeOp(GT_MUL_LONG, type, op1, op2)
+    GenTreeMultiRegOp(var_types type, GenTreePtr op1, GenTreePtr op2)
+        : GenTreeOp(GT_MUL_LONG, type, op1, op2), gtOtherReg(REG_NA)
     {
```

Comment:
I would assume that, if you changing this from `GenTreeMulLong` to `GenTreeMultiRegOp`, you expect to use this more generally than just for `GT_MUL_LONG`. If that's not the case, then this should be kept as `GenTreeMulLong`. If that is the case, then you shouldn't have an implicit oper on the constructor.